### ISSUE TITLE
[Lookup Anything] throws an exception when looking up something in the crafting page

### ISSUE THREAD

**Issue by tstaples** [Pathoschild/StardewMods#40]
This only occurs when looking up something in the "standalone" crafting page, the only one I know of being the cooking interface via your kitchen. The exception is:

```
[17:45:29 TRACE LookupAnything] Received a lookup request... searching the open 'CraftingPage' menu... an error occurred.
[17:45:29 ERROR LookupAnything] Something went wrong looking that up:
System.ArgumentNullException: Value cannot be null.
Parameter name: value
   at System.Runtime.Caching.MemoryCacheEntry..ctor(String key, Object value, DateTimeOffset absExp, TimeSpan slidingExp, CacheItemPriority priority, Collection`1 dependencies, CacheEntryRemovedCallback removedCallback, MemoryCache cache)
   at System.Runtime.Caching.MemoryCache.AddOrGetExistingInternal(String key, Object value, CacheItemPolicy policy)
   at System.Runtime.Caching.MemoryCache.AddOrGetExisting(String key, Object value, CacheItemPolicy policy, String regionName)
   at System.Runtime.Caching.ObjectCache.Add(String key, Object value, CacheItemPolicy policy, String regionName)
   at StardewModdingAPI.Framework.Reflection.ReflectionHelper.GetCached[TMemberInfo](String key, Func`1 fetch) in D:\source\_Stardew\SMAPI\src\StardewModdingAPI\Framework\Reflection\ReflectionHelper.cs:line 235
   at StardewModdingAPI.Framework.Reflection.ReflectionHelper.GetFieldFromHierarchy[TValue](Type type, Object obj, String name, BindingFlags bindingFlags) in D:\source\_Stardew\SMAPI\src\StardewModdingAPI\Framework\Reflection\ReflectionHelper.cs:line 168
   at StardewModdingAPI.Framework.Reflection.ReflectionHelper.GetPrivateField[TValue](Object obj, String name, Boolean required) in D:\source\_Stardew\SMAPI\src\StardewModdingAPI\Framework\Reflection\ReflectionHelper.cs:line 41
   at StardewModdingAPI.Framework.Reflection.ReflectionHelper.GetPrivateValue[TValue](Object obj, String name, Boolean required) in D:\source\_Stardew\SMAPI\src\StardewModdingAPI\Framework\Reflection\ReflectionHelper.cs:line 73
   at Pathoschild.Stardew.LookupAnything.Framework.TargetFactory.GetSubjectFrom(IClickableMenu menu, Vector2 cursorPosition) in D:\source\_Stardew\LookupAnything\LookupAnything\Framework\TargetFactory.cs:line 362
   at Pathoschild.Stardew.LookupAnything.LookupAnythingMod.GetSubject(StringBuilder logMessage, LookupMode lookupMode) in D:\source\_Stardew\LookupAnything\LookupAnything\LookupAnythingMod.cs:line 341
   at Pathoschild.Stardew.LookupAnything.LookupAnythingMod.<>c__DisplayClass17_0.<ShowLookup>b__0() in D:\source\_Stardew\LookupAnything\LookupAnything\LookupAnythingMod.cs:line 322
   at Pathoschild.Stardew.LookupAnything.GameHelper.InterceptErrors(IMonitor monitor, String verb, String detailedVerb, Action action, Action`1 onError) in D:\source\_Stardew\LookupAnything\LookupAnything\GameHelper.cs:line 357
```

**Comment by Pathoschild** [comment]
This behaves much more nicely in SMAPI 1.5, which fixes the reflection API issue:
>![image](https://cloud.githubusercontent.com/assets/230581/21446705/cc62902a-c897-11e6-8e8d-bfa39f8bf648.png)

I'll add support for the standalone crafting page in the 1.10 release.

**Comment by Pathoschild** [comment]
Fixed in `develop` for the upcoming 1.10 release. :)
